### PR TITLE
LPS-198250 Set FF to RELEASED for 'Schedule KB article'

### PR DESCRIPTION
**Pull request by marcogalluzzi**
https://liferay.atlassian.net/browse/LPS-198250

### CODE REVIEW

**Comment by marcogalluzzi** [comment]
Setting this PR to `on hold` until #5017 is merged into master.

**Review by adolfopa — APPROVED** [review]

**Comment by marcogalluzzi** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
ci:forward:force

**Pull request closed without merging**
